### PR TITLE
Adding changes of commit 23ef8f6

### PR DESCRIPTION
**Pull request by AnkithB2020**
Commit 23ef8f6 includes changes related to replacement of
os.mkfs() with os.fsformat()

Link explaining the replacement details:
https://forum.pycom.io/topic/3351/new-development-firmware-release-v1-19-0-b1/2

Issue: https://forum.pycom.io/topic/5606/what-happened-to-pin_deepsleep_wakeup/9

### CODE REVIEW

File: content/tutorials/lte/firmware.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Please start with the following steps:
 2. Select Firmware Type `stable` in the communication window to upgrade to version `v1.18.1.r1`
 
 
-The modem firmware files are password protected, to download them you should be a forum.pycom.io member and access to: 
+The modem firmware files are password protected, to download them you should be a forum.pycom.io member and access to:
```
Comment (lilycey) [inline content/tutorials/lte/firmware.md]: 1st comment: The modem firmware files are password protected. In order to download them, head to  forum.pycom.io and become a member (if you aren't already) and click on:

File: content/tutorials/lte/firmware.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Please start with the following steps:
...
 Announcements & News --&gt;  Announcements only for members --&gt; Firmware Files for Sequans LTE modem now are secured, or clicking [Here](https://forum.pycom.io/topic/4020/firmware-files-for-sequans-lte-modem-now-are-secured).
```
Comment (lilycey) [inline content/tutorials/lte/firmware.md]: 2nd comment: Announcements & News --&gt;  Announcements for members only --&gt;  the Firmware Files for the Sequans LTE modem are now secured or click [here](https://forum.pycom.io/topic/4020/firmware-files-for-sequans-lte-modem-now-are-secured).

File: content/tutorials/lte/firmware.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Please start with the following steps:
...
 
 You can find the different versions of firmwares available [here](https://software.pycom.io/downloads/sequans2.html).
```
Comment (lilycey) [inline content/tutorials/lte/firmware.md]: 3rd comment: You can find the different available firmware versions [here](https://software.pycom.io/downloads/sequans2.html).

File: content/tutorials/lte/firmware.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ import sqnsupgrade
 sqnsupgrade.run('upgdiff_38638-to-39529.dup')
 ```
 If you are updating the Sequans firmware on your module for the first time, please use instead the file upgdiff_33080-to-39529.dup (1.2M) from the same archive.
```
Comment (lilycey) [inline content/tutorials/lte/firmware.md]: 4th comment: If you are updating the Sequans firmware on your module for the first time, please use this file instead  upgdiff_33080-to-39529.dup (1.2M), which is from the same archive.

File: content/tutorials/lte/firmware.md [hunk 5]
```diff
@@ -57,7 +57,7 @@ import sqnsupgrade
...
-Similar upgrade packages are available for the NB-IoT firmwares. 
+Similar upgrade packages are available for the NB-IoT firmwares.
```
Comment (lilycey) [inline content/tutorials/lte/firmware.md]: 5th: Similar upgrade packages are available for the NB-IoT firmware.

File: content/tutorials/lte/firmware.md [hunk 6]
```diff
@@ -267,5 +270,3 @@ The latest version of the `sqnsupgrade` class has a few additional features that
 
 #### sqnsupgrade.run(load_fff=True)
     New optional command line option load_fff for the sqnsupgrade.run() command. This is designed to be an internal flag. And should only applied when advised by pycom support.
```
Comment (lilycey) [inline content/tutorials/lte/firmware.md]: 6th: There is an optional command line load_fff for the sqnsupgrade.run() command. This is designed to be an internal flag. IMPORTANT: This should only be used when advised by Pycom Support.

**Review by lilycey — COMMENTED** [review]
Updated to improve readability

**Comment by jirkadev** [comment]
@AnkithB2020 I think you need to open also a second PR

File: content/tutorials/lte/firmware.md (hunk 1 above)
Comment (AnkithB2020) [reply content/tutorials/lte/firmware.md]: Comment incorporated.

File: content/tutorials/lte/firmware.md (hunk 2 above)
Comment (AnkithB2020) [reply content/tutorials/lte/firmware.md]: Comment incorporated.

Comment (AnkithB2020) [reply content/tutorials/lte/firmware.md]: Comment incorporated.

File: content/tutorials/lte/firmware.md [hunk 7]
```diff
@@ -37,10 +37,10 @@ Please start with the following steps:
 2. Select Firmware Type `stable` in the communication window to upgrade to version `v1.18.1.r1`
 
 
-The modem firmware files are password protected, to download them you should be a forum.pycom.io member and access to: 
-Announcements & News --&gt;  Announcements only for members --&gt; Firmware Files for Sequans LTE modem now are secured, or clicking [Here](https://forum.pycom.io/topic/4020/firmware-files-for-sequans-lte-modem-now-are-secured).
+The modem firmware files are password protected. In order to download them, head to forum.pycom.io and become a member (if you aren't already) and click on:
+Announcements & News --> Announcements for members only --> the Firmware Files for the Sequans LTE modem are now secured or click here.
 
-You can find the different versions of firmwares available [here](https://software.pycom.io/downloads/sequans2.html).
+You can find the different available firmware versions here.
```
Comment (jirkadev) [inline content/tutorials/lte/firmware.md]: We need to keep the URL for that markdown link there 

`[here](https://software.pycom.io/downloads/sequans2.html)`

**Review by jirkadev — COMMENTED** [review]
Thank you for your changes. Please remove .DS_Store files from your commit.

File: content/tutorials/lte/firmware.md (hunk 7 above)
Comment (AnkithB2020) [reply content/tutorials/lte/firmware.md]: Ok, sure, I have kept the URL for the markdown link now.

Comment (AnkithB2020) [reply content/tutorials/lte/firmware.md]: Also, .DS_Store file has been removed.

**Comment by jirkadev** [comment]
I still see DS_store files here in this PR. If you go to "files changed" you would see them too

File: content/tutorials/lte/firmware.md (hunk 7 above)
Comment (jirkadev) [reply content/tutorials/lte/firmware.md]: Actually, the Here links were not fixed consider how it looks in your PR and in production docs.

Please install and use hugo on your pc and `hugo serve` to preview your changes locally

Screen Shot 2020-01-29 at 14 16 33
Screen Shot 2020-01-29 at 14 17 01

**Review by Xykon — DISMISSED** [review]

**Review by Xykon — APPROVED** [review]

**Pull request merged**